src/main/kotlin/com/hj/leetcode/kotlin/problem330/Solution.kt

```diff
@@ -0,0 +1,28 @@
+package com.hj.leetcode.kotlin.problem330
+
+import kotlin.math.min
+
+/**
+ * LeetCode page: [330. Patching Array](https://leetcode.com/problems/patching-array/);
+ */
+class Solution {
+    /* Complexity:
+     * Time O(M+Log(n)) and Space O(1) where M is the size of nums;
+     */
+    fun minPatches(nums: IntArray, n: Int): Int {
+        var result = 0
+        var index = 0
+        var covered = 0
+
+        while (covered < n) {
+            if (index < nums.size && nums[index] <= covered + 1) {
+                covered += nums[index]
```

Comment:
Use min(...) to guard against overflow, although it is not expected to occur under the current problem constraints.

Reply:
Addressed by https://github.com/hj-core/hj-leetcode-kotlin/pull/378/commits/d7df18960bf0e14507d63fe6d62bdacb75f07d73